src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts

```diff
@@ -181,6 +174,98 @@ class OZWNodeConfig extends LitElement {
     `;
   }
 
+  private _generate_config_block(item) {
+    return html` <ha-card class="content" .value=${item.value}>
+      <div class="card-content">
+        <b>${item.label}</b><br />
+        <span class="secondary">${item.help}</span>
+        ${["Byte", "Short", "Int"].includes(item.type)
+          ? html` <paper-input
+              type="number"
+              .value=${item.value}
+              .max=${item.max}
+              .min=${item.min}
+              ?disabled=${item.parameter === 999}
+              .parameter=${item.parameter}
+            >
+            </paper-input>`
```

Comment:
Can't we refactor this in the backend so we can just use `ha-form`?

Reply:
I'm not familiar with `ha-form` or it's workings, is there a good resource to read up on that?

Reply:
Let's talk on discord